### PR TITLE
Fix background colors in App Bars #230

### PR DESCRIPTION
**Pull request by PrakashIrom**
I made some changes in the Material Theme and added containerColor parameter for the NavigationBar to fix the background color issue, but for the TopAppBar I don't see any differences in the screenshots you uploaded.
If there's any changes you want to make or anything wrong with what I did, please do tell me.

### CODE REVIEW

File: app/src/main/java/com/lorenzovainigli/foodexpirationdates/ui/theme/Color.kt [hunk 1]
```diff
@@ -6,7 +6,7 @@ val md_theme_light_primary = Color(0xFF934B00)
 val md_theme_light_onPrimary = Color(0xFFFFFFFF)
 val md_theme_light_primaryContainer = Color(0xFFFFDCC5)
 val md_theme_light_onPrimaryContainer = Color(0xFF301400)
-val md_theme_light_secondary = Color(0xFF755945)
+val md_theme_light_secondary = Color(0xFF755945)//
```
Comment (lorenzovngl) [inline app/src/main/java/com/lorenzovainigli/foodexpirationdates/ui/theme/Color.kt:9]: Has this been included in the commit by mistake?

File: app/src/main/java/com/lorenzovainigli/foodexpirationdates/ui/theme/Color.kt [hunk 2]
```diff
@@ -23,7 +23,7 @@ val md_theme_light_onBackground = Color(0xFF201A17)
 val md_theme_light_surface = Color(0xFFFFFBFF)
 val md_theme_light_onSurface = Color(0xFF201A17)
 val md_theme_light_surfaceVariant = Color(0xFFF3DFD2)
-val md_theme_light_onSurfaceVariant = Color(0xFF52443B)
+val md_theme_light_onSurfaceVariant = Color(0xFF52443B)//
```
Comment (lorenzovngl) [inline app/src/main/java/com/lorenzovainigli/foodexpirationdates/ui/theme/Color.kt:26]: Has this been included in the commit by mistake?

File: app/src/main/java/com/lorenzovainigli/foodexpirationdates/ui/theme/Color.kt [hunk 3]
```diff
@@ -38,7 +38,7 @@ val md_theme_dark_onPrimary = Color(0xFF4F2500)
 val md_theme_dark_primaryContainer = Color(0xFF703800)
 val md_theme_dark_onPrimaryContainer = Color(0xFFFFDCC5)
 val md_theme_dark_secondary = Color(0xFFE4BFA7)
-val md_theme_dark_onSecondary = Color(0xFF422B1A)
+val md_theme_dark_onSecondary = Color(0xFF422B1A)//
```
Comment (lorenzovngl) [inline app/src/main/java/com/lorenzovainigli/foodexpirationdates/ui/theme/Color.kt:41]: Has this been included in the commit by mistake?

File: app/src/main/java/com/lorenzovainigli/foodexpirationdates/ui/theme/Theme.kt [hunk 4]
```diff
@@ -63,7 +63,7 @@ private val DarkColors = darkColorScheme(
     surfaceVariant = md_theme_dark_surfaceVariant,
     onSurfaceVariant = md_theme_dark_onSurfaceVariant,
     outline = md_theme_dark_outline,
-    inverseOnSurface = md_theme_dark_inverseOnSurface,
+    inverseOnSurface = md_theme_light_inverseSurface,
```
Comment (lorenzovngl) [inline app/src/main/java/com/lorenzovainigli/foodexpirationdates/ui/theme/Theme.kt:66]: Please don't modify this file. Also, this is incorrect: why assigning a light-themed color to a dark-themed variable?

File: app/src/main/java/com/lorenzovainigli/foodexpirationdates/view/composable/MyBottomAppBar.kt [hunk 5]
```diff
@@ -48,7 +49,7 @@ fun MyBottomAppBar(
         unselectedIcon = Icons.Outlined.Info
     )
     )
-    NavigationBar {
+    NavigationBar(containerColor = MaterialTheme.colorScheme.inverseOnSurface) {
```
Comment (lorenzovngl) [inline app/src/main/java/com/lorenzovainigli/foodexpirationdates/view/composable/MyBottomAppBar.kt:52]: It's correct to change `containerColor`, but please assign the following value:
```suggestion
    NavigationBar(
        containerColor = MaterialTheme.colorScheme.surfaceColorAtElevation(TonalElevation.level2())
    ) {
```

**Review by lorenzovngl — CHANGES_REQUESTED** [review]
It was more difficult than expected to figure out why this bug has appeared.

Regarding TopAppBar, you're right. I'm sorry for not providing a meaningful screenshot. Please fix it as described here:

https://github.com/lorenzovngl/FoodExpirationDates/blob/acb33a312aeefe5c811a0b826dfe428ca5e4fd2c/app/src/main/java/com/lorenzovainigli/foodexpirationdates/view/composable/MyTopAppBar.kt#L51-L53

```diff
        navigationIcon = navigationIcon,
-       scrollBehavior = scrollBehavior
+       scrollBehavior = scrollBehavior,
+       colors = TopAppBarDefaults.topAppBarColors(
+            scrolledContainerColor = MaterialTheme.colorScheme.surfaceColorAtElevation(TonalElevation.level2())
+       )
    )
}
```

File: app/src/main/java/com/lorenzovainigli/foodexpirationdates/ui/theme/Color.kt (hunk 3 above)
Comment (PrakashIrom) [reply app/src/main/java/com/lorenzovainigli/foodexpirationdates/ui/theme/Color.kt:41]: ohh yeah sorry it was by mistake

File: app/src/main/java/com/lorenzovainigli/foodexpirationdates/ui/theme/Theme.kt (hunk 4 above)
Comment (PrakashIrom) [reply app/src/main/java/com/lorenzovainigli/foodexpirationdates/ui/theme/Theme.kt:66]: ohh my bad I was changing the color to see the matching bottom bar color.

File: app/src/main/java/com/lorenzovainigli/foodexpirationdates/view/composable/MyBottomAppBar.kt (hunk 5 above)
Comment (PrakashIrom) [reply app/src/main/java/com/lorenzovainigli/foodexpirationdates/view/composable/MyBottomAppBar.kt:52]: ok I will check

**Comment by PrakashIrom** [comment]
Okay I will do it.

**Comment by PrakashIrom** [comment]
I have made the changes please do check

**Review by lorenzovngl — APPROVED** [review]

**Comment by lorenzovngl** [comment]
All good! Thank you for your efforts and welcome among the contributors of Food Expiration Dates! 🎉

**Comment by PrakashIrom** [comment]
Thank you @lorenzovngl once again, without your help I couldn't do.

**Pull request merged**